t/01-hash.t

```diff
@@ -66,10 +66,10 @@ is_deeply(
 );
 
 is_deeply(
-    {slice_grep { $_ gt 'a' } \%h},
```

Comment:
Damit wird der Test für https://github.com/perl5-utils/Hash-MoreUtils/blob/master/lib/Hash/MoreUtils.pm#L99 in die andere Richtung gekippt.
Ein klügeres %h und ein grep, das z.B. nur Schlüssellängen von 2 durchlässt, wäre vielleicht eine hübsche Idee.